File: Python/account.py
from mailbox import BabylMessage
from unittest.util import sorted_list_difference
from xml.dom.minidom import Document


class Account:
    id       = int
    name     = str
    document = str
    email    = str
    password = str

    def __init__(self, name, document):
        self.name     = name
        self.document = document

File: JS/Account.js
class Account {

    constructor(name, document, email, password) {
        this.id;
        this.name = name;
        this.document = document;
        this.email = email;
        this.password = password;
    }

    printDataAccount() {
        console.table("The name: " + this.name);
        console.table("The document: " + this.document);
        console.table("The email: " + this.email);
        console.table("The password: " + this.password);
    }

}